src/app/services/startup/startup.service.ts

```diff
@@ -169,10 +177,62 @@ export class ArlasStartupService {
     }
 
     public setConfigService(data) {
+        /**First set the raw config data in order to create an ArlasExploreApi instance */
+        const newConfig = this.configUpdater(data);
+        this.configService.setConfig(newConfig);
+        this.collaborativesearchService.setFetchOptions(this.fetchOptions);
+        const arlasUrl = this.configService.getValue('arlas.server.url');
+        const configuration: Configuration = new Configuration();
+        this.arlasExploreApi = new ArlasExploreApi(
+            configuration,
+            arlasUrl,
+            portableFetch
+        );
+        this.collaborativesearchService.setConfigService(this.configService);
+        this.collaborativesearchService.setExploreApi(this.arlasExploreApi);
```

Comment:
There is a bug, that I just discovered here

data is an array of 2 elements: the configobject and the language

we should do data[0] = newConfig to take into account the changes of `configUpdater`